payjoin/src/send/multi_party/mod.rs

```diff
@@ -0,0 +1,176 @@
+use bitcoin::{FeeRate, Psbt};
+use error::{
+    CreateRequestError, FinalizeResponseError, FinalizedError, InternalCreateRequestError,
+    InternalFinalizeResponseError, InternalFinalizedError,
+};
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use super::v2::{self, extract_request, EncapsulationError, HpkeContext};
+use super::{BuildSenderError, PsbtContext};
+use crate::ohttp::ohttp_decapsulate;
+use crate::receive::ImplementationError;
+use crate::send::v2::{serialize_v2_body, V2PostContext};
+use crate::uri::UrlExt;
+use crate::{PjUri, Request};
+
+mod error;
+
+#[derive(Clone)]
+pub struct SenderBuilder<'a>(v2::SenderBuilder<'a>);
+
+impl<'a> SenderBuilder<'a> {
+    pub fn new(psbt: Psbt, uri: PjUri<'a>) -> Self { Self(v2::SenderBuilder::new(psbt, uri)) }
+    pub fn build_recommended(self, min_fee_rate: FeeRate) -> Result<Sender, BuildSenderError> {
+        let v2 = v2::SenderBuilder::new(self.0 .0.psbt, self.0 .0.uri)
```

Comment:
These spaces ("0. 0") are added after auto-formatting :angry:

Reply:
An Alias would work. I would say lets wait till a payjoin cli integration to see what is most ergonomic. Anything at this point for a experimental feature is a pre-optimization.

I've always thought the most ergonomic option is actually to start with a v2::Sender / SenderBuilder. If the feature flag is on you can `build_with_optimistic_merge`. You can think of this as an upgrade. Or you always start with the multiparty sender and downgrade to v2. The former makes more sense to me.